spec/reek/smells/nested_iterators_spec.rb

```diff
@@ -271,6 +271,15 @@ def bad(fred)
       expect(src).to reek_of(:NestedIterators, count: 2).with_config(config)
     end
   end
+
+  it 'should analyze safe-navigation-operator without error' do
+    src = '
+      def f
+        x&.tap {|_| }
+      end
+    '
+    expect(src).not_to reek_of(:NestedIterators, count: 2)
```

Comment:
When the analysis raises an error (without the fix), this spec fails as `F`. However, I don't think this `expect` is good for one in my mind.

I want something like:

```rb
  expect {
    # run reek against src
  }.not_to raise_error
```

Any advice? (or is this good enough?)